CHANGELOG.md

```diff
@@ -574,7 +574,7 @@ current changes on git with [previous release tags][git_tag_comparison].
 - [Automatic batching/instancing of draw commands][9685]
 - [Directly copy data into uniform buffers][9865]
 - [Allow other plugins to create renderer resources][9925]
-- [Use EntityHashMap<Entity, T> for render world entity storage for better performance][9903]
+- [Use EntityHashMap<T> for render world entity storage for better performance][9903]
```

Comment:
This seems to be the changelog for the last stable release which I don't think should be changed.

Reply:
I just did a big search and replace, will fix now.